apps/web/src/views/Info/components/InfoNav/index.tsx

```diff
@@ -3,28 +3,28 @@ import {
   ButtonMenu,
   ButtonMenuItem,
   Flex,
+  Message,
+  MessageText,
+  Text,
   UserMenu,
   UserMenuDivider,
   UserMenuItem,
-  Text,
-  Message,
-  MessageText,
 } from '@pancakeswap/uikit'
 import { NextLinkFromReactRouter } from '@pancakeswap/widgets-internal'
 
-import { useCallback, useMemo } from 'react'
-import {} from 'hooks/useSwitchNetwork'
 import { ChainId } from '@pancakeswap/chains'
 import { useTranslation } from '@pancakeswap/localization'
+import { ChainLogo } from 'components/Logo/ChainLogo'
+import { ASSET_CDN } from 'config/constants/endpoints'
+import {} from 'hooks/useSwitchNetwork'
```

Comment:
<img src="https://media1.giphy.com/media/iSuffsM2olusE/giphy.gif"/>
I'm not sure what is imported here? 😅

Reply:
yup `import {} from 'hooks/useSwitchNetwork'` should be removed ^^